src/main/java/org/mockito/verification/VerificationMode.java

```diff
@@ -30,8 +29,11 @@
  * See examples in javadoc for {@link Mockito#verify(Object, VerificationMode)}
  */
 public interface VerificationMode {
-    
-    void verify(VerificationData data);
+
+    /**
+     * Performs the verification
+     */
+    void verify(org.mockito.internal.verification.api.VerificationData data);
```

Comment:
Right, this type will be made public (breaking change) in the next major version of Mockito, see  #744. It's not really feasible to stop leaking private API without introducing incompatible changes.